tests/server_test_cases/person_note_tests.py

```diff
@@ -2850,29 +2850,35 @@ def test_sms_altogether_invalid(self):
             phone_number='+10987654321',
             path='/global/api/handle_sms?key=sms_key&lang=en',
             expected_response_code=400,
-            expected_response='The given receiver_phone_number is not found in '
-                 'sms_number_to_repo config.')
+            expected_response= 'You&#39;ve reached Person Finder, but '
+                'there&#39;s not a repository assigned for +10987654321.')
```

Comment:
Skroob: "That's amazing! I've got the same combination on my luggage!"

Reply:
Hahaha.